lesson_07/dags/process_sales.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+from airflow.models import Variable
+from airflow import DAG
+from airflow.providers.http.operators.http import SimpleHttpOperator
+import json
+
+DEFAULT_ARGS = {
+    'depends_on_past': False,
+    'email': ['admin@example.com'],
+    'email_on_failure': True,
+    'email_on_retry': False,
+    'retries': 2,
+    'retry_delay': 30,
+}
+
+DATE = "{{ execution_date | ds }}"
+BASE_DIR = Variable.get('BASE_DIR_DOWNLOAD_LESSON_07')
```

Comment:
this will ping the Airflow database each time the schedule run the file, so it's better to use other approaches (config files etc)
Note that using Variables outside of Operators is kinda an antipattern in Airflow